craft_cli/helptexts.py

```diff
@@ -326,19 +353,23 @@ def _build_plain_command_help(
             see_also_block.extend(("    " + name) for name in sorted(other_command_names))
             textblocks.append("\n".join(see_also_block))
 
-        # footer
+        # help for all commands
         textblocks.append(
             f"""
-            For a summary of all commands, run '{self.appname} help --all'.
-        """
+            For a summary of all commands, run '{self.appname} help --all'."""
         )
 
+        # documentation link
+        if self._docs_base_url:
+            command_url = f"{self._docs_base_url}/reference/commands/{command.name}"
+            textblocks[-1] += f"\nFor more information, check out: {command_url}"
```

Comment:
The reason I'm appending to the previous one is because I think it looks better to keep the 2 or 3 "for more info..." sentences within the same block. 

A:
<details>
```
Usage:
    testcraft [help] <command>

Summary:    Verify your stuff

Global options:
                 -h, --help:  Show this help message and exit
              -v, --verbose:  Show debug information and be more verbose
                -q, --quiet:  Only show warnings and errors, not
                              progress
                --verbosity:  Set the verbosity level to 'quiet',
                              'brief', 'verbose', 'debug' or 'trace'
              -V, --version:  Show the application version and exit

Starter commands:
                       init:  Create an initial project filetree
                    version:  Show the application version and exit

Commands can be classified as follows:
                  Lifecycle:  build, clean, pack, prime, pull, stage
                      Other:  init, version
    Test Store management:  ...

For more information about a command, run testcraft help <command>'.
For a summary of all commands, run 'testcraft help --all'.
For more information about testcraft, check out: https://canonical-testcraft.readthedocs-hosted.com/en/latest
```
</details>


B:
<details>
```
Usage:
    testcraft [help] <command>

Summary:    Verify your stuff

Global options:
                 -h, --help:  Show this help message and exit
              -v, --verbose:  Show debug information and be more verbose
                -q, --quiet:  Only show warnings and errors, not
                              progress
                --verbosity:  Set the verbosity level to 'quiet',
                              'brief', 'verbose', 'debug' or 'trace'
              -V, --version:  Show the application version and exit

Starter commands:
                       init:  Create an initial project filetree
                    version:  Show the application version and exit

Commands can be classified as follows:
                  Lifecycle:  build, clean, pack, prime, pull, stage
                      Other:  init, version
    Test Store management:  ...

For more information about a command, run testcraft help <command>'.
For a summary of all commands, run 'testcraft help --all'.

For more information about testcraft, check out: https://canonical-testcraft.readthedocs-hosted.com/en/latest
```
</details>

Reply:
I did improve the readability as you suggested.

Reply:
I agree, with the readability thing. Thank you for the explaination